newjersey/navigator.business.nj.gov
# feat: [#188251182] Add essential questions to landlord industry

## Description

Adds two essential questions to the landlord industry, one dependent on the other



### Ticket



This pull request resolves [#188251182](https://www.pivotaltracker.com/story/show/188251182).

### Approach



### Steps to Test
Start a new business in the landlord industry
A new essential question should pop up. If you choose "Short Term Rental" no follow up should appear. If you choose either of the other options a second non-essential question should appear.

If you answer "Short Term" to the first question, you should see the short term rental task in roadmap step 4
If you answer "Long Term" to the first question and "Yes" to the second, you should see the multiple dwelling registration task in step 4
If you answer "Long Term" to the first question and "No" to the second, you should see landlord registration task and smoke detector task
If you answer "Both" to the first question and "Yes" to the second, you should see short term rental task and multiple dwelling registration task
If you answer "Both" to the first question and "No" to the second, you should see landlord registration task, smoke detector task, and short term rental task




### Notes



## Code author checklist
